modules/ROOT/pages/schema/syntax.adoc

```diff
@@ -0,0 +1,630 @@
+:description: Syntax for how to manage graph types and constraints in Neo4j.
+:table-caption!:
+= Syntax
+
+This page contains the syntax for the graph types and constraints commands.
+More details about the syntax can be found in the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/syntax/#administration-syntax-reading[Operations Manual -> Cypher syntax for administration commands].
+
+[role=label--cypher-25-only label--enterprise-edition label--new-Neo4j-2025.xx]
+[[graph-types]]
+== Graph types
+
+[[element-types]]
+=== Node and relationship element types
+
+.Syntax for node element types, <node element type>
+[source, syntax]
+----
+([alias] {IS | :} IdentifyingLabelName {=> | IMPLIES} [{IS | :} ImpliedLabelName[&...]] [“{“ {propertyName [[IS] :: | IS TYPED] <TYPE> [IS [NODE] {KEY | UNIQUE}]}[, ...] “}”])
+[REQUIRE {alias.propertyName | (alias.propertyName[, ...])} IS [NODE] {KEY | UNIQUE}]
+----
+
+[NOTE]
+A node element type requires either one or more implied labels or a non-empty property map.
+
+For more information, see xref:schema/graph-types/set-graph-types.adoc#node-element-types[Set graph types -> Node element types].
+
+.Syntax for relationship element types, <relationship element type>
+[source, syntax]
+-----
+([nodeAlias | [nodeAlias] {IS | :} LabelName [=> | IMPLIES]])-”[“ [alias] {IS | :} IDENTIFYING_RELATIONSHIP_TYPE {=> | IMPLIES} [“{“ {propertyName [[IS] :: | IS TYPED] <TYPE> [IS [REL[ATIONSHIP]] {KEY | UNIQUE}]}[, ...] “}”] “]”->([nodeAlias | [nodeAlias] {IS | :} LabelName [=> | IMPLIES]])
+[REQUIRE {alias.propertyName | (alias.propertyName[, ...])} IS [REL[ATIONSHIP]] {KEY | UNIQUE}]
+-----
+
+[NOTE]
+====
+A relationship element type requires either:
+
+* A non-empty source or a target node.
+* A non-empty property map.
+====
+
+For more information, see xref:schema/graph-types/set-graph-types.adoc#relationship-element-types[Set graph types -> Relationship element types].
+
+The three variations of the property type expression, `IS ::`, `::`, and `IS TYPED` are syntactic synonyms for the same expression.
```

Comment:
Might be better to move up the element type specific parts to the elements section, part 1
```suggestion
`<TYPE>` can be one of the property types listed in *add link to [[constraints-definitions]]*, the `NOT NULL` versions of those types or `ANY NOT NULL`.

The three variations of the property type expression, `IS ::`, `::`, and `IS TYPED` are syntactic synonyms for the same expression.
```